runbotics/runbotics-scheduler/src/scheduler-database/process/process.service.ts

```diff
@@ -57,7 +55,7 @@ export class ProcessService {
         return this.processRepository.update({ id: process.id }, process);
     }
 
-    async checkAccessAndGetById(processId: number, user: UserEntity) {
+    async checkAccessAndGetById(processId: number, user: User) {
         // TODO: to be updated after process migration to scheduler
         // Explanation: Process privileges should be based on process collections
         // and checking privileges should be adjusted
```

Comment:
it is still something we need to wait to do? processes were already migrated to scheduler

Reply:
yes, so that should be done in process collection task